.travis.yml

```diff
@@ -87,14 +89,10 @@ before_script:
     - sed -i "s/yourusernamehere/root/" wp-tests-config.php
     - sed -i "s/yourpasswordhere//" wp-tests-config.php
     - cd "/tmp/wordpress/src/wp-content/plugins/$PLUGIN_SLUG"
-    - gem install sass
-    - gem install compass
-    - nvm install 5
-    - npm install -g npm@'3.8.9'
-    - npm install -g gulp-cli
-    - npm install
+    - ./tests/load-travis.sh
 
```

Comment:
You can inline the contents of `tests/load-travis.sh` here if you'd rather not create a new file:
```shell
- |
  if [ "$WP_TRAVISCI" != "phpunit" ]; then
	gem install sass
	gem install compass
	nvm install 5
	npm install -g npm@'3.8.9'
	npm install -g gulp-cli
	npm install
 fi
```